app/fetcher/src/CommentRender.hs

```diff
@@ -161,12 +161,14 @@ genBuildFailuresTable
 
     gen_matched_build_section idx (CommitBuilds.BuildWithLogContext (CommitBuilds.NewCommitBuild (Builds.StorableBuild (DbHelpers.WithId ubuild_id universal_build) build_obj) match_obj _ _) (CommitBuilds.LogContext _ log_lines)) = [
         M.heading 4 $ T.unwords [
-            circleci_image_link
+            "<details>"
+          , "<summary>"
```

Comment:
This requires a corresponding `</summary>` closing tag, right?  It could go in as the last item in this list.

Reply:
Updated with a corresponding closing `</summary>` tag